Makefile

```diff
@@ -77,7 +77,7 @@ ENABLE_KEYLOCK                   := 1
 
 TARGET = firmware
 
-GIT_HASH_TMP := $(shell git rev-parse --short HEAD)
```

Comment:
On the official OEFW, this is actually 7.

Reply:
Hence the python script has also to be changed (as it checks for a maxlen of 10).